samples/nrf9160/modem_shell/prj.conf

```diff
@@ -12,9 +12,10 @@ CONFIG_MOSH_CURL=y
 CONFIG_MOSH_LINK=y
 CONFIG_MOSH_SMS=y
 CONFIG_MOSH_GNSS=y
-# By default Low Noise Amplifier (LNA) is not enabled, generally this should be enabled if the
-# on board GPS antenna is used. The LNA can also be controlled using shell commands.
-#CONFIG_MOSH_GNSS_ENABLE_LNA=y
+
+# Choose appropriate GNSS antenna configuration (one only)
+CONFIG_MOSH_GNSS_ANTENNA_EXTERNAL=y
+#CONFIG_MOSH_GNSS_ANTENNA_ONBOARD=y
```

Comment:
There's no need to set the already default value. Actually it would cause a warning when CONFIG_MOSH_GNSS=n, so it's better to remove it.
```suggestion
# Uncomment to use external antenna for GNSS
#CONFIG_MOSH_GNSS_ANTENNA_EXTERNAL=y
```